docs/reference/commandline/dockerd.md

```diff
@@ -992,6 +992,21 @@ with user namespaces enabled or not. If the daemon is configured with user
 namespaces, the Security Options entry in the response will list "userns" as
 one of the enabled security features.
 
+> **Note**:
+> Due to the need to segregate content in the Docker daemon’s local cache of layer
+> data by the mappings provided, the daemon uses *separate local caches* for when
+> user namespaces are "on" and "off". When running the docker daemon with `--userns-remap`,
+> the root of your graph directory (/var/lib/docker by default) will have one additional level
+> of indirection, correlating to the remapped root uid and gid. For example, if the
+> remapping user provided to the `--userns-remap` flag has subordinate user and group
+> ranges that begin with ID 10000, then the root of the graph directory for all images
+> and containers running with that remap setting will reside in `/var/lib/docker/10000.10000`.
```

Comment:
This example is technically correct, but it might be helpful to use a "canonical" example that matches what people will probably experience.  Given ID 10000 is still in the first 64K that is traditionally left alone for the default ID space of the host, most distros are going to start the offset at 100000 (which is just a nice round number above the 64K initial range).  Again, this is minor as technically you could use 10000, but we wouldn't want to encourage that in case people are building their own `/etc/sub{u,g}id` files on a customized distro base.